### ISSUE TITLE
The dock has the top of icons cut off during the bouncing animation

### ISSUE THREAD

**Issue by CuriousCoconut** [elementary/browser#151]
### What Happened?

This is happening with the stable and daily builds of OS8. When I click on an app in the dock and it does the bouncing animation the top of the icon disappears at the top of the bounce.

### Steps to Reproduce

Click on an icon in the dock and watch the top of the icon get cut off while it bounces.

### Expected Behavior

The icon should bounce without the top of it getting cut off.

### OS Version

8.x (Early Access)

### Session Type

Classic Session (X11, This is the default)

### Software Version

Latest release (I have run all updates)

**Comment by ryonakano** [comment]
Duplicate of https://github.com/elementary/dock/issues/275

**Comment by ryonakano** [comment]
Please do not post the same issue by yourself: you're also reporting this at https://github.com/elementary/dock/issues/312.

**Comment by CuriousCoconut** [comment]
Yeah because I accidentally posted it in the wrong place (the feedback app
isn't working by the way) and I couldn't figure out how to delete it.

Sorry for the inconvenience.

**Comment by ryonakano** [comment]
Ah I got it. You can't "delete" an issue (unless you're a maintainer of the repository) but instead you can close it and that's fine :+1:

(FYI it would be helpful for other people to write why you close the issue when closing it, e.g.: `Closing since I posted to a wrong repository`)
